ui/src/app/apps/apps.service.ts

```diff
@@ -96,7 +96,8 @@ export class AppsService {
 
     return this.http.delete(AppsService.appsUrl + '/' + appRegistration.type + '/' + appRegistration.name, options)
       .map(data => {
-        this.appRegistrations.items = this.appRegistrations.items.filter(item => item.name !== appRegistration.name);
+        const index: number = this.appRegistrations.items.findIndex(item => item.name === appRegistration.name);
+        this.appRegistrations.items.splice(index, 1);
```

Comment:
@cppwfs Pleas create an issue for that one -- I need to think about that one a bit. When on the same page and you delete things, I am not doing page refreshes for single item deletes, I think for absolutely perfect and consistent behavior that needs to change, and we need to do a server-call to refresh the page.